lib/std/Thread.zig

```diff
@@ -79,7 +79,7 @@ pub fn setName(self: Thread, name: []const u8) SetNameError!void {
             var buf: [32]u8 = undefined;
             const path = try std.fmt.bufPrint(&buf, "/proc/self/task/{d}/comm", .{self.getHandle()});
 
-            const file = try std.fs.cwd().openFile(path, .{ .write = true });
+            const file = try std.fs.cwd().openFile(path, .{ .mode = .read_write });
```

Comment:
This seems to only actually want `.mode = .write`

Reply:
I assumed that it wants both because `.read` is `true` by default, as per this commit:
https://github.com/ziglang/zig/blob/85de022c5671d777f62ddff254a814dab05242fc/lib/std/fs/file.zig#L75-L77
Are you implying that the existing code is wrong? Will check a bit more on it.